wire-ios/Wire-iOS/Sources/UserInterface/Availability/AvailabilityStringBuilder.swift

```diff
@@ -74,6 +79,12 @@ enum AvailabilityStringBuilder {
             interactive: false,
             color: textColor
         )
+        if appendYouSuffix, !L10n.Localizable.UserCell.Title.youSuffix.isEmpty {
```

Comment:
Is it possible that` L10n.Localizable.UserCell.Title.youSuffix` isEmpty?

Reply:
Who knows what strings all the localisation files contain ...
I removed it since the implementation works with an empty string as well.